src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/EmployeeController.php

```diff
@@ -486,6 +492,16 @@ protected function getErrorMessages(Exception $e)
                 'The object cannot be loaded (or found)',
                 'Admin.Notifications.Error'
             ),
+            UploadedImageConstraintException::class => [
+                UploadedImageConstraintException::EXCEEDED_SIZE => $this->trans(
+                    'Max file size allowed is "%d" bytes.', 'Admin.Notifications.Error',
+                    [$iniConfig->getUploadMaxSizeInBytes()]
+                ),
+                UploadedImageConstraintException::UNRECOGNIZED_FORMAT => $this->trans(
+                    'Image format not recognized, allowed formats are: .gif, .jpg, .png',
```

Comment:
```suggestion
                    'Image format not recognized, allowed formats are: .%s',
```

Should it be this? formats have to came from a const? What if we authorized .jpeg, or .svg or .bmp?

Reply:
Thank you @PierreRambaud, I agree with you + it will be only 1 string to translate for translators.

Reply:
Hmm I simply reused string that has beend widely used in other places, but okay I will refactor so available formats are a const instead.